mono/mini/debugger-agent.c

```diff
@@ -10620,15 +10621,43 @@ void burst_mono_install_hooks(BurstMonoDebuggerCallbacks* callbacks, void* domai
 #endif /* DISABLE_SDB */
 }
 
-// Called from various threads
+// In Old editors the behaviour is that burst_mono_update_tracking_pointers is called from various threads,
+//after a request from the burst managed debugger dll that the managed debugger domain needs to be torn
+//down and re-created to cause the debugger clients to refresh the information they hold about the domain.
+
+// In Newer editors, this is now only called once during initialisation, these pointers are fixed for the 
+//lifetime of the editor once installed.
 void burst_mono_update_tracking_pointers(MonoDomain* domain, MonoClass* klass)
 {
 #ifndef DISABLE_SDB
 	burst_lock();
 	g_BurstKlass = klass;
 	g_BurstDebugDomain = domain;
+	MonoAssembly* assembly=m_class_get_image (g_BurstKlass)->assembly;
```

Comment:
```suggestion
	MonoAssembly* assembly = m_class_get_image (g_BurstKlass)->assembly;
```
Sorry for the nit, just to keep things consistent with the rest of the function.